locales/en.json

```diff
@@ -302,7 +303,7 @@
     "notificationError": "Could not open QR code scanner: {error}."
   },
   "QRGenerator": {
-    "bodyPaymentNoteInvalid": "You cannot write special symbols like {}<>^ and more than 100 characters.",
+    "bodyPaymentNoteInvalid": "You cannot write special symbols like {}<br>^ and more than 100 characters.",
```

Comment:
I think it should be "or" not "and" grammatically. We might as well correct it here